src/qibo/models/encodings.py

```diff
@@ -458,7 +471,13 @@ def _sparse_encoder_farias(data, nqubits: int, backend=None, **kwargs):
 
 
 def binary_encoder(
-    data, parametrization: str = "hyperspherical", backend=None, **kwargs
+    data,
+    parametrization: str = "hyperspherical",
+    nqubits: int = None,
+    backend=None,
+    codewords=None,
+    keep_antictrls: bool = False,
```

Comment:
```suggestion
    codewords=None,
    keep_antictrls: bool = False,
    backend=None,
```

Description of new arguments is missing in the Args of the docstring.